rook/scripts/wait-for-rook.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+here="$(dirname "$(readlink -f "$0")")"
+echo -n "Testing if rook is ready up to 10 times..."
+for i in {1..5}
+  do
+    echo -n " ${i}"
+    failure=0
+    if ! bash "${here}/test-rook.sh" "$1" &> /dev/null; then
+      failure=1
+    fi
+    # If no failures, we are ready to move on
+    if [ ${failure} -eq 0 ]; then
+      echo
+      echo "Rook tests passed - running again to produce output"
+      bash "${here}/test-rook.sh" "$1"
+      exit 0
+    fi
+    sleep 60
+done
+if [[ ! ${failure} -eq 0 ]]; then
```

Comment:
Is this necessary? If you've gotten here it needs to have failed. Otherwise if it hasn't it would've been caught in the if statement in the for loop and then exited. Or is it some other corner case I'm missing

```suggestion
```

Reply:
I think you are correct